packages/renderer/src/lib/container/ContainerList.svelte

```diff
@@ -463,204 +485,15 @@ function setStoppedFilter() {
-                      <div class="flex flex-nowrap text-xs font-extra-light text-gray-900 items-center">
-                        <div>{container.state}</div>
-                        <div class="pl-2 pr-2 inline-flex">{container.displayPort}</div>
-                      </div>
-                    </div>
-                  </div>
-                </td>
-                <td class="pl-3 whitespace-nowrap hover:cursor-pointer group">
-                  <div class="flex items-center text-xs p-1 rounded-md text-gray-500">
-                    <ProviderInfo provider="{container.engineType}" context="{container.engineId}" />
-                  </div>
-                </td>
-                <!-- Open the container details, TODO: open image details instead? -->
-                <td
-                  class="pl-3 whitespace-nowrap hover:cursor-pointer group"
-                  on:click="{() => openDetailsContainer(container)}">
-                  <div class="flex items-center">
-                    <div class="text-sm text-gray-700 overflow-hidden text-ellipsis" title="{container.image}">
-                      {container.shortImage}
-                    </div>
-                  </div></td>
-                <td class="whitespace-nowrap pl-3">
-                  <div class="flex items-center">
-                    <div class="text-sm text-gray-700">
-                      <StateChange state="{container.state}">{container.uptime}</StateChange>
-                    </div>
-                  </div>
-                </td>
-                <td
-                  class="pl-6 text-right whitespace-nowrap {containerGroup.type === ContainerGroupInfoTypeUI.STANDALONE
-                    ? 'rounded-tr-lg'
-                    : ''} {index === containerGroup.containers.length - 1 ? 'rounded-br-lg' : ''}">
-                  <div class="flex w-full">
-                    <div class="flex items-center w-5">
-                      {#if container.actionError}
-                        <ErrorMessage error="{container.actionError}" icon />
-                      {:else}
-                        <div>&nbsp;</div>
-                      {/if}
-                    </div>
-                    <div class="text-right w-full mr-[5px]">
-                      <ContainerActions
-                        container="{container}"
-                        dropdownMenu="{true}"
-                        on:update="{() => (containerGroups = [...containerGroups])}" />
-                    </div>
-                  </div>
-                </td>
-              </tr>
-            {/each}
-          {/if}
-          <tr><td class="leading-[8px]">&nbsp;</td></tr>
-        {/each}
-      </tbody>
-    </table>
+    <Table
+      kind="container"
+      bind:this="{table}"
+      bind:selectedItemsNumber="{selectedItemsNumber}"
+      data="{containerGroups}"
+      columns="{columns}"
+      row="{row}"
+      on:update="{() => (containerGroups = [...containerGroups])}">
```

Comment:
should we add `defaultSortColumn="Name"` ?

Reply:
Done in latest update.